src/shared_modules/utils/flatbuffers/schemas/common/common_agentInfo.fbs

```diff
@@ -0,0 +1,4 @@
+table AgentInfo {
```

Comment:
Table names in other files follow the snakecase format. Why is this one different?

Reply:
Ditto